[tool:pytest]
testpaths = test
# pytest with coverage seems to be causing issues in parallel tests
# addopts = --cov=lyncs_io --cov-report term-missing
